dm/master/coordinator/coordinator.go

```diff
@@ -499,7 +499,7 @@ func (c *Coordinator) restartMysqlTask(w *Worker, cfg *config.MysqlConfig) bool
 		ectx, cancel := context.WithTimeout(c.etcdCli.Ctx(), etcdTimeout)
 		defer cancel()
 		resp, err := c.etcdCli.Get(ectx, common.WorkerKeepAliveKeyAdapter.Encode(w.Address(), w.Name()))
-		if err != nil {
+		if err == nil {
```

Comment:
How do we handle `err != nil`?

Reply:
`err != nil` means because of some reasons we can't get the keepAlive result for this worker. I think add retry may refine this problem, but it can't solve this problem. It's hard to solve this problem in current mechanism.

Reply:
So, how about just logging it for debugging now?

Reply:
OK, I will add a log.